metals/src/main/resources/db/migration/V3__Jar_symbols.sql

```diff
@@ -0,0 +1,17 @@
+-- Do not allow more than one combination of path and digest
+create table indexed_jar(
+  id int auto_increment,
+  path varchar not null,
+  last_modified bigint not null,
+  size bigint not null,
+  primary key (path, last_modified, size)
+);
+-- Allow for multiple jars with same symbols and paths
+create table toplevel_symbol(
+  symbol varchar not null,
+  path varchar not null,
+  jar int,
```

Comment:
is this a foreign key on `indexed_jar -> id`?

Reply:
Yes, indeed. Should I add a foreign key reference?

Reply:
I generally prefer to add foreign key references, so that there's no ambiguity when reading the schema. I would add it 👍

Reply:
Added foreign key restraint from toplevel_symbol to indexed_jar.